NEWS.adoc

```diff
@@ -15,6 +15,7 @@ Bug fixes:
  - Fix cursor behaviour during staging. (#842, #1028)
  - Fix navigation in split tree view.
  - Enable textconv in the stage view.
+ - Change the blame view to use porcelain. (#978, #1189)
```

Comment:
I'm not sure if porcelain vs. incremental is relevant for users. I'd probably say something like

The blame view now honors the textconv option and blame flags like -L.